sunpy/coordinates/tests/test_transformations.py

```diff
@@ -5,10 +5,49 @@
 from astropy.coordinates import SkyCoord, get_body_barycentric
 from astropy.time import Time
 
-from sunpy.coordinates import Helioprojective, HeliographicStonyhurst, HeliographicCarrington, get_sun_L0
+import sunpy.coordinates.transformations as trans
+from sunpy.coordinates import (Helioprojective, HeliographicStonyhurst,
+                               HeliographicCarrington, get_sun_L0,
+                               Heliocentric)
 from sunpy.time import parse_time
 
 
+def test_new_hcc_to_hgs():
+    # Generate 10 random coordinates
+    for i in range(10):
+        x = np.random.rand() * u.km
+        y = np.random.rand() * u.km
+        z = np.random.rand() * u.km
```

Comment:
Are you sure that these coordinates are valid? `np.random.rand` only gives you values between `[0,1)`, so all these 10 coordinates you are checking are in a small subset... that may be pretty much closed enough for the transformation to don't find any error.